rts/lra/lra-test/pom.xml

```diff
@@ -75,7 +80,17 @@
                 </property>
             </activation>
             <properties>
-                <lra.coordinator.trace.params>-Dthorntail.logging=TRACE</lra.coordinator.trace.params>
+<!--                # Disabling console logging-->
+                <lra.coordinator.log.console.disable>-Dquarkus.log.console.enable=false</lra.coordinator.log.console.disable>
+<!--                # Enabling file logging-->
+                <lra.coordinator.log.file.enable>-Dquarkus.log.file.enable=true</lra.coordinator.log.file.enable>
+                <lra.coordinator.log.file.level>-Dquarkus.log.file.level=TRACE</lra.coordinator.log.file.level>
+<!--                # Send output to a trace.log file under the target/failsafe-reports/ directory-->
+                <lra.coordinator.log.file.path>-Dquarkus.log.file.path=target/failsafe-reports/trace.log</lra.coordinator.log.file.path>
```

Comment:
please rename the `trace.log` to `lra-coordinator-trace.log` or similar

Reply:
will do the same